modules/ROOT/pages/database/database-connector-get-started.adoc

```diff
@@ -0,0 +1,88 @@
+= Database Connector - Get Started - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Support Category: Select
+
+Database Connector Version 1.5.4
+
+The Database connector establishes communication between your Mule app and a relational database.
+
+After adding the connector to Studio, the connector and its operations appear in the Mule Palette.
+
+== Database Listeners and Operations
+
+The Database connector provides a listener to read from a database in the Source section of a flow.
+You can execute other operations which also allow you to read and write to a database anywhere in the Process section.
+For example, instead of writing single records to a database, bulk operations allow you to modify a collection of records by simply supplying the SQL to modify each record. 
+Other operations allow you to carry out Data Definition Language (DDL) operations, to execute stored procedures, or to execute entire SQL scripts at once.
+
+== About Connectors
+
+Anypoint connectors are Mule runtime engine extensions that enable you to connect
+to APIs and resources on external systems, such as Salesforce, Database, ServiceNow,
+and Twitter.
+
+== Prerequisites
+
+Before creating an app, you must have access to the Database target resource and
+Anypoint Platform. You must also understand how to create a Mule app using
+Design Center or Anypoint Studio.
+
+== Supported Database Types
+
+The Database Connector provides specialized support for the following database:
+
+* MySQL
+* Oracle
+* Microsoft SQL Server
+
+You can set up other JDBC databases using a generic JDBC configuration. You can also reference a JDBC `DataSource` object or a `XADataSource` object, which is required for XA transactions. You typically create the object in Studio using Spring.
+
+== Audience
+
+* Starting user -- Read this Get Started document and create your Mule app in
+xref:db/database-connector-design-center.adoc[Design Center] or
+xref:db/database-connector-studio.adoc[Anypoint Studio].
+The xref:db/database-connector-config-topics.adoc[Additional Configuration Information] topic
```

Comment:
Do you have additional configurations? I didn't see any. If not, you'll need to change this.

Reply:
No, I do not.

Reply:
Removed the link from the list.